.script/tests/KqlvalidationsTests/CustomTables/Ubiquiti.json

```diff
@@ -145,9 +145,13 @@
             "Name": "EventResultDetails",
             "Type": "String"
         },
-        {
-            "Name": "Message",
-            "Type": "String"
-        }
+      {
+        "Name": "Message",
+        "Type": "String"
+      },
+      {
+        "Name": "DnsQueryName",
```

Comment:
I could not find the field DnsQueryName in the [parser](https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Ubiquiti%20UniFi/Parsers/UbiquitiAuditEvent.txt)

Reply:
Yes, @oshezaf I checked and removed it and made other changes too in accordance with the respective parsers, Thanks!